po/LINGUAS

```diff
@@ -7,3 +7,4 @@ oc
 sv
 tr
 ur
+zh_CN
```

Comment:
Can you please add a newline character at the end of the last line?

Reply:
Sure. added